cmux_test.go

```diff
@@ -439,6 +511,7 @@ func interestingGoroutines() (gs []string) {
 		}
 
 		if stack == "" ||
+			strings.Contains(stack, "main.main()") ||
```

Comment:
i'm not sure if this match is correct, but without this the tests do not work in go 1.7